doc/whats_new/v0.23.rst

```diff
@@ -339,6 +339,11 @@ Changelog
   computing statistics when calling `partial_fit` on sparse inputs.
   :pr:`16466` by :user:`Guillaume Lemaitre <glemaitre>`.
 
+- |Fix| Fix a bug in :class:`preprocessing.Normalizer` which was not taking the
+  absolute value of the maximum values before normalizing the vectors.
+  :pr:`16632` by :user:`Maura Pintor <Maupin1991>`
+  and :user:`Battista Biggio <freezebat>`.
```

Comment:
my username is: bbiggio
@Maupin1991 can you fix it? thanks!

Reply:
Done! :smile:

Reply:
Did @bbigio contribute to the solution?

Reply:
It has been a joint effort. We work in the same laboratory and we've been working on this issue together.

Reply:
Great!